src/jsonforms-test-utils.tsx

```diff
@@ -0,0 +1,39 @@
+import { vi } from "vitest"
+import { JsonForms } from "@jsonforms/react"
+import { render as RTLrender } from "@testing-library/react"
+import { Form } from "antd"
+import { cellRegistryEntries, rendererRegistryEntries } from "./renderers"
+import { JSONSchema } from "json-schema-to-ts"
+import { UISchema } from "./ui-schema"
+import { JsonSchema7 } from "@jsonforms/core"
+
+global.matchMedia =
+  global.matchMedia ||
+  function () {
+    return {
+      addListener: vi.fn(),
+      removeListener: vi.fn(),
+    }
+  }
+
+type RenderProps<T> = {
+  schema: JSONSchema
+  data?: T
+  uischema?: UISchema
+  onChange?: (result: { data: T }) => void
+}
+
+export function render<T>({ schema, data, uischema, onChange }: RenderProps<T>): ReturnType<typeof RTLrender>{
```

Comment:
Is this moved? I thought this already existed in the repo?

Reply:
No, I didn't see it anywhere. I think I found it now though...

Reply:
Used the existing function, but had to make some tweaks.